NDTensors/src/diag/tensoralgebra/contract.jl

```diff
@@ -58,6 +58,28 @@ function _contract!!(
   return R
 end
 
+function contract!(
+  output_tensor::Exposed{<:AbstractArray,<:DiagTensor},
+  labelsoutput_tensor,
+  tensor1::Exposed,
+  labelstensor1,
+  tensor2::Exposed,
+  labelstensor2,
+  α::Number=one(Bool),
+  β::Number=zero(Bool),
+)
+  @assert α == one(Bool)
+  @assert β == zero(Bool)
```

Comment:
```suggestion
  @assert isone(α)
  @assert iszero(β)
```
The same thing but simpler and easier to read. I like to use `isone` and `iszero` when appropriate.